docs/cds/cds.rst

```diff
@@ -0,0 +1,59 @@
+
+.. doctest-skip-all
+.. _astroquery.cds:
+
+
+**********************************
+CDS MOC Service (`astroquery.cds`)
+**********************************
+
+Getting started
+===============
+
+This module provides a python interface for querying the `CDS MOC Service`_ from the `Centre de Données de Strasbourg (CDS)`_.
+
+The `CDS MOC Server`_ tool aims at retrieving as fast as possible the list of astronomical data sets
+(catalogs, surveys, ...) having at least one observation in a specific sky region.
+The default result is an ID list but one can ask for different output formats such as:
+
+* The number of the data sets.
+* A dictionary containing the lists of meta data for all the resulting data sets. These lists of meta data are each indexed by the ID of the data set they are referring to.
+* A MOC resulting from the union/intersection of all the MOCs of the resulting data sets.
```

Comment:
have you defined the acronym MOC yet?  I don't see it.

Reply:
Ah, it's defined below.  Best define it here, though: Multi-Order Coverage object?